docs/pages/variables.md

```diff
@@ -18,67 +20,64 @@ on other platforms, for that reason we add a variable `GIT_COMMIT_SHA` which alw
 point to the SHA of a commit, no matter the underlying platform, as long as the platform
 supplies a predefined variables with the SHA.
 
-
```

Comment:
I see that you have removed all escaping from the table here. I guess that is still fine at this point and we don't really need it right this second. However, this does not relate to the commit that you have created and should be split up into a separate commit. (this goes for all documentation formatting changes)

Reply:
IDE actually made the removal, when formatted the table to nice looking version. Have to to the nice layout probably by hand and not by VScode format.

Reply:
Updated all, let see if PR would realize that also at some point...